app/form_models/publishers/job_listing/applying_for_the_job_form.rb

```diff
@@ -4,7 +4,7 @@ class Publishers::JobListing::ApplyingForTheJobForm < Publishers::JobListing::Va
   validates :enable_job_applications, inclusion: { in: [true, false, "true", "false"] }
 
   def self.fields
-    %i[enable_job_applications]
+    %i[personal_statement_guidance enable_job_applications]
```

Comment:
We could remove `personal_statement_guidance` because it's not needed for new listings and it's not editable for published legacy listings, and draft legacy listings aren't editable at all